pkg/serviceaccount/openidmetadata_test.go

```diff
@@ -226,7 +226,9 @@ func TestURLBoundaries(t *testing.T) {
 		WantOK bool
 	}{
 		{"OIDC config path", "/.well-known/openid-configuration", true},
+		{"OIDC config path with trailing slash", "/.well-known/openid-configuration/", true},
 		{"JWKS path", "/openid/v1/jwks", true},
+		{"JWKS path with trailing slash", "/openid/v1/jwks/", true},
 		{"well-known", "/.well-known", false},
 		{"subpath", "/openid/v1/jwks/foo", false},
 		{"query", "/openid/v1/jwks?format=yaml", true},
```

Comment:
can you update this test to make sure we're not following redirects, so we're actually getting the expected response with and without the slash (rather than a redirect to the other URL)

Reply:
good point, added that to the unit test